clang/test/SemaCXX/warn-thread-safety-analysis.cpp

```diff
@@ -4955,13 +4968,18 @@ class Foo {
     //showDataCell(*datap2_); // xpected-warning {{reading the value pointed to by 'datap2_' requires holding mutex 'mu_'}}
 
     int a = data_[0];       // expected-warning {{reading variable 'data_' requires holding mutex 'mu_'}}
+
+    (void)&data_ao_;      // expected-warning {{reading variable 'data_ao_' requires holding mutex 'mu_'}}
+    showData(&data_ao_);  // expected-warning {{reading variable 'data_ao_' requires holding mutex 'mu_'}}
```

Comment:
This is probably not so relevant. This test exists to call `const` overloads of operators, but you only defined one overload.

Reply:
Added the second overload.